public/styles/components/nav-primary/_nav-utility.scss

```diff
@@ -47,11 +47,23 @@
       }
     }
   }
+
+  .bsds-nav-logo-secondary {
+    width: auto;
+    max-height: space(4x);
```

Comment:
we'll need to set max-height on the primary logo too
you can use `https://www.cdx.bostonscientific.com/content/dam/BSC%20Cardiac%20Diagnostics%20Inc-Wordmark%20(3).svg` in storybook to test